Networking/Networking/Remote/ShippingLabelRemote.swift

```diff
@@ -230,23 +227,12 @@ public final class ShippingLabelRemote: Remote, ShippingLabelRemoteProtocol {
     /// - Parameters:
     ///     - siteID: Remote ID of the site.
     ///     - orderID: Remote ID of the order that owns the shipping labels.
-    ///     - canCreatePaymentMethod: Whether the client supports creating new payment methods.
-    ///     - canCreateCustomsForm: Whether the client supports creating customs forms.
-    ///     - canCreatePackage: Whether the client supports creating packages.
     ///     - completion: Closure to be executed upon completion.
     public func checkCreationEligibility(siteID: Int64,
                                          orderID: Int64,
-                                         canCreatePaymentMethod: Bool,
-                                         canCreateCustomsForm: Bool,
-                                         canCreatePackage: Bool,
                                          completion: @escaping (Result<ShippingLabelCreationEligibilityResponse, Error>) -> Void) {
-        let parameters = [
-            ParameterKey.canCreatePaymentMethod: canCreatePaymentMethod,
-            ParameterKey.canCreateCustomsForm: canCreateCustomsForm,
-            ParameterKey.canCreatePackage: canCreatePackage
```

Comment:
nit: I think the keys can also be deleted 😄

Reply:
Removed in fadfc736b62d1b283c30d318342ed83b169fdc03.